### PR TITLE
ViewItem,  where the description is shown

### PR DESCRIPTION
**Pull request by DanielRBech**
## Description

Created the view activity page, where the description is shown

## Type of PR

- Feature

## Related Issues/Tickets

https://github.com/aau-giraf/weekplan/issues/9

## Checklist

- [x] Code follows the code guidelines
- [x] Tests have been added/updated (if applicable)
- [x] Documentation has been updated (if applicable)

### CODE REVIEW

**Comment by DanielRBech** [comment]
Forgot the pull branch

File: app/viewitem.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return (
+        <View style={styles.container}>
+            {data && (
+                <View style={styles.activityContainer}>
+                    <Text style={styles.activityName}>{data.name}</Text>
+                    <Text style={styles.activityDescription}>{data.description}</Text>
+                    <Text style={styles.activityTime}>{data.startTime} - {data.endTime}</Text>
+                </View>
+            )}
+            <Button title='Go Back' onPress={handlePress} />
```
Comment (MertzAndreas) [inline app/viewitem.tsx:32]: This is wrong use href

**Review by MertzAndreas — CHANGES_REQUESTED** [review]
Almost there

File: hooks/useActivity.ts [hunk 2]
```diff
@@ -156,3 +157,15 @@ export default function useActivity({ date }: { date: Date }) {
...
+
+  const useFetchActivity = useQuery<ActivityDTO>({
+    queryFn: () => fetchActivityRequest(activityId),
+    queryKey: ['activity', activityId],
+  });
+
+  return {
+    useFetchActivity,
+  };
+}
```
Comment (MertzAndreas) [inline hooks/useActivity.ts:171]: Error handling?

**Review by MertzAndreas — CHANGES_REQUESTED** [review]

File: app/viewitem.tsx (hunk 1 above)
Comment (DanielRBech) [reply app/viewitem.tsx:32]: Hvis du bare mener at lave the til router.back i stedet så er det gjort

**Review by MaksimHamko — APPROVED** [review]
Very good work <3

**Pull request merged**